### ISSUE TITLE
Missing python2.7 on the newest RaspberryOS- dataplicity automatic installer script doesn't work

### ISSUE THREAD

**Issue by shorti1996** [wildfoundry/dataplicity-agent#74]
**tl;dr;** there's no info in case there's no python 2.7 in the system

I don't know if this is a good place to raise the issue but I just want to let you guys know that on the latest RaspberryOS (2022-01-28-raspios-bullseye-armhf-lite) there's no python2.7 and there's no info that it needs to be installed separately. Basically, Debian 11 no longer contains python2 by default. [[here](https://wiki.debian.org/Python)]
As a result, a typical user who doesn't look for documentation online or doesn't check logs doesn't know why it's not working and probably will just give up.
Simple `apt-get install python2` is sufficient but the installer script could at least suggest that.

**Comment by Radoslaw-K** [comment]
Hey @shorti1996 thank you for letting us know, as it happens we're working on a fix for it right now 👍

**Comment by saeedjohar** [comment]
Is it fixed now? or any update?

**Comment by shorti1996** [comment]
You can just install python 2.7 in your system manually. It will do the trick.

**Comment by Radoslaw-K** [comment]
> Is it fixed now? or any update? Moreover it seem to install v0.4.34 of Dataplicity agent

We're getting closer to having a release ready 👍 still some work to do .

**Comment by Radoslaw-K** [comment]
New release is now available ☺ fully compatible with Python3 💯

**Comment by saeedjohar** [comment]
Thank you for the update. Installed it without any issue. 

Not sure if I should be mentioning it here; 
When I click on my device, the screen is blank. Am I missing anything? 

![image](https://user-images.githubusercontent.com/19384084/162317045-035bd077-56e7-4398-851a-0b96caa16bbe.png)

**Comment by Radoslaw-K** [comment]
> Thank you for the update. Installed it without any issue.
> 
> Not sure if I should be mentioning it here; When I click on my device, the screen is blank. Am I missing anything?
> 
> ![image](https://user-images.githubusercontent.com/19384084/162317045-035bd077-56e7-4398-851a-0b96caa16bbe.png)

Which operating system is running on your Raspberry Pi ? 
and
Does the log `/var/log/dataplicity.log` file show any errors ?

**Comment by saeedjohar** [comment]
My RPi 4 Details: 
```lsb_release -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 11 (bullseye)
Release:	11
Codename:	bullseye
```

The output of the `/var/log/dataplicity.log`
```[14/Apr/2022 12:00:29] agent	: dataplicity 0.5.8
[14/Apr/2022 12:00:29] agent	: python executable=/usr/bin/python
[14/Apr/2022 12:00:29] agent	: python version=3.9.2 (default, Mar 12 2021, 04:06:34)  [GCC 10.2.1 20210110]
[14/Apr/2022 12:00:29] agent	: uname=Linux rpidev 5.15.32-v7l+ #1538 SMP Thu Mar 31 19:39:41 BST 2022 armv7l 
[14/Apr/2022 12:00:29] agent	: remote_directory=/home/dataplicity/remote
[14/Apr/2022 12:00:29] agent	: m2m=wss://m2m.dataplicity.com/m2m/?features=scan
[14/Apr/2022 12:00:29] agent	: api=https://api.dataplicity.com/
[14/Apr/2022 12:00:29] agent	: serial=xxxxxxxxxxxxxxxxx
[14/Apr/2022 12:00:29] agent	: poll=60
[14/Apr/2022 12:00:29] agent	: disk poll failed unable to contact JSONRPC server 'https://api.dataplicity.com/' (<ur
lopen error [Errno -3] Temporary failure in name resolution>)
[14/Apr/2022 12:00:29] agent	: sync failed unable to contact JSONRPC server 'https://api.dataplicity.com/' (<urlopen
 error [Errno -3] Temporary failure in name resolution>)
```

**Comment by Radoslaw-K** [comment]
Apologies for the late response :( 

It would seem like the agent is okay . The failure in name resolution would lead me to think this is something to do with the internet connection on the device or there was some temporary problem on our servers . 
Is the internet on the device fully working ? 
Does `ping google.com` work ?
Does the agent work now ?

**Comment by saeedjohar** [comment]
Yes, the device has internet. I am already working with this device. 
I can ping google.com but I can't ping api.dataplicity.com
